src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -1,25 +1,15 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-import { NavLink, Outlet } from 'react-router-dom';
+import { Navigate, Outlet } from 'react-router-dom';
 import RoutesList from '../../navigation/routes';
 import { selectUser } from '../../store/auth/authSlice';
 
-export default function DashboardPage() {
+function DashboardPage() {
   const userInfo = useSelector(selectUser);
-
-  // show unauthorized screen if no user is found in redux store
-  if (!userInfo.isLoggedIn) {
-    return (
-      <div>
-        <h1>Unauthorized :(</h1>
-        <span>
-          <NavLink to={RoutesList.Login}>Login</NavLink>
-          to gain access
-        </span>
-      </div>
-    );
-  }
-
-  // returns child route elements
-  return <Outlet />;
+  return (
+    <div>
```

Comment:
do we need this div ?

Reply:
error  Fragments should contain more than one child - otherwise, there’s no need for a Fragment at all  react/jsx-no-useless-fragment

I have this error if I do not have div

Reply:
so do not use any fragment or div

Reply:
adding a change